ush/templates/FV3LAM_wflow.xml

```diff
@@ -379,13 +403,24 @@ MODULES_RUN_TASK_FP script.
          {%- elif machine in ["ORION"]  %}
          <datadep age="00:00:05:00"><cyclestr>{{ extrn_mdl_sysbasedir_ics }}/gdas.@Y@m@d/@H/atmos/gdas.t@Hz.atmf{{ "%03d" % h }}.nc</cyclestr></datadep>
          {%- else %}
+         {%- if extrn_mdl_name_lbcs in ["GEFS"] %}
+         <datadep age="00:00:05:00"> <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}{{ input_ensmem_subdir }}/@y@j@H000{{ "%03d" % h }}</cyclestr></datadep>
+         {%- else %}
          <datadep age="00:00:05:00"> <cyclestr offset="-{{ extrn_mdl_lbcs_offset_hrs }}:00:00">{{ extrn_mdl_sysbasedir_lbcs }}/@y@j@H000{{ "%03d" % h }}</cyclestr></datadep>
          {%- endif %}
+         {%- endif %}
          {%- endfor %}
        </and>
     </dependency>
 
   </task>
+
+{% if do_ensemble %}
+ </metatask>
+{% endif %}
+
+
+{% if not do_ensemble %}
```

Comment:
You can set <cycledef group="boundary_long"> 00 01 01 01 2100 * </cycledef> to skip the boundary_long task. Hence we don't need an extra "{% if not do_ensemble %}" logic here.

Reply:
Removed quite a few " {% if not do_ensemble %}".